open_instruct/rejection_sampling/api_generate.py

```diff
@@ -6,13 +6,17 @@
 from typing import List, Optional
 
 from openai import AsyncOpenAI
-from prompt_templates import get_generation_template, get_judgment_template
 from tqdm.asyncio import tqdm
 
+from open_instruct.rejection_sampling.prompt_templates import (
+    get_generation_template,
+    get_judgment_template,
+)
+
 
 @dataclass
 class LLMGenerationConfig:
-    n: int = 64
+    num_completions: int = 64
```

Comment:
I like the name changes! To be more consistent with the codebase, I changed it to `num_completions`